modules/olm-updating-metallb-operatorgroup.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * operators/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+[id="olm-updating-metallb-operatorgroup_{context}"]
+= Editing the MetalLB Operator Operator group
+
+When upgrading from any MetalLB Operator version up to and including 4.10 to 4.11 and later, remove `spec.targetNamespaces` from the Operator group custom resource (CR). You must remove the spec regardless of whether you used the web console or the CLI to delete the MetalLB Operator.
+[NOTE]
+====
+The MetalLB Operator version 4.11 or later only supports the `AllNamespaces` install mode, whereas 4.10 or earlier versions support `OwnNamespace` or `SingleNamespace` modes.
+====
+
+.Prerequisites
+
+- You have access to an {product-title} cluster with `cluster-admin` permissions.
+- You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. List the Operator groups in the `metallb-system` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc get operatorgroup -n metallb-system
+----
++
+.Example output
+
+[source,terminal]
+----
+NAME                   AGE
+metallb-system-7jc66   85m
+----
+
+. Verify that the `spec.targetNamespaces` is present in the Operator group CR associated with the `metallb-system` namespace by running the following command:
```

Comment:
Not enough to hold up a merge review, but we should probably add a noun after this config field: 

```suggestion
. Verify that the `spec.targetNamespaces` environment variable is present in the Operator group CR associated with the `metallb-system` namespace by running the following command:
```